src/Framework/FieldsAPI/Date.php

```diff
@@ -15,4 +16,29 @@ class Date extends Field
     use Concerns\HasDescription;
 
     const TYPE = 'date';
+
+    /** @var string */
+    protected $dateFormat = '';
```

Comment:
@pauloiankoski do you think we should add a default format instead of an empty string?

Reply:
Added a default value reenforcing the same default value we have set for the template: `yyyy/mm/dd`

Reply:
Cool, that makes sense.  Should we do the same for `phoneFormat`?

Reply:
Nothing on `phoneFormat` is already an option. In the template, it'll renders a native `<input type="phone" />`.